azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingFragment.kt

```diff
@@ -49,6 +50,7 @@ internal class CallingFragment :
     private lateinit var participantListView: ParticipantListView
     private lateinit var bannerView: BannerView
     private lateinit var lobbyOverlay: LobbyOverlayView
+    private lateinit var holdOverlay: HoldOverlayView
```

Comment:
Are you adding another overlay, instead of reusing the lobby overlay?

Reply:
I added another Overlay to not abuse the Lobby.

Single responsibility principal, it's just a fork of Lobby with UI changes and different trigger for visibility (callState = LOCAL_HOLD)